caasp-389-ds-image/root/usr/local/bin/entrypoint.sh

```diff
@@ -27,38 +27,69 @@ if [ ! -e "/data/config/setup.inf" ]; then
 
     echo -e "\n>> Creating localhost configuration...\n"
     cat <<EOT >> /data/config/setup.inf
-[General]
-FullMachineName=localhost.${DOMAIN_SUFFIX}
-;SuiteSpotUserID=dirsrv
-;SuiteSpotGroup=dirsrv
-;Should not run as root
-SuiteSpotUserID=root
-;Should not run as root
-SuiteSpotGroup=root
-StrictHostCheck=false
+[general]
+config_version = 2
+;defaults = 999999999
```

Comment:
It seems many keys here in general, slapd are commented out? What is the purpose here? If no need, why we need to put here then comment out?

It's more like some testing purpose but left in this commit?

Reply:
I create this from CLI 'dscreate create-template'. I keep these variables, so I know what is default value. It can be removed.